src/helpers/EmailProof.ts

```diff
@@ -60,15 +60,16 @@ export default class EmailProof extends BaseProof implements EmailProofSchema {
   async generateInput(
     domain: string,
     signature: string,
-    eddsaPublicKey: PublicKey,
-    nullifierSignature: string
+    eddsaPublicKey: PublicKey
   ) {
     const maxDomainLength = 90
     const messageUInt8 = this.padZeroesOnRightUint8(
       utils.toUtf8Bytes(domain),
       maxDomainLength
     )
-    const { r: r2, s: s2 } = utils.splitSignature(nullifierSignature)
+    const randomNumbersArray = crypto.getRandomValues(new BigUint64Array(2))
```

Comment:
How about doing it like this? ```const [r2, s2] = crypto.getRandomValues(new BigUint64Array(2))```

Reply:
I also noticed that we have a file that isn't used anywhere `unpackECDSASignature.ts`

Reply:
> How about doing it like this? `const [r2, s2] = crypto.getRandomValues(new BigUint64Array(2))`

Same issue as above, though I wish that this worked since it's a really clean solution.

Reply:
> I also noticed that we have a file that isn't used anywhere `unpackECDSASignature.ts`

Deleted the file, I didn't find any references in `seal-cred-frontend`. Thanks

Reply:
Nvm it works